src/tokenizer/tokenizer.py

```diff
@@ -698,7 +698,7 @@ def html_escape(match):
     return unicode_chr(int(g[1:]))
 
 
-def gen_from_string(txt, replace_composite_glyphs=True, replace_html_escapes=False):
+def gen_from_string(txt, replace_composite_glyphs=True, replace_html_escapes=False, one_sent_per_line=False):
```

Comment:
Note that `replace_composite_glyphs` is by default `True`. The option, if present, should therefore probably set it to `False`, and its meaning should be reversed (i.e. "Do not replace composite glyphs" or "Leave composite glyphs unchanged"). This may apply to other options as well - check what the defaults are, and the command line options should reverse those defaults, if present.

Reply:
Changed to "keep_composite_glyphs", only option that defaults to True.